react/ProductSummaryTypes.ts

```diff
@@ -33,6 +33,13 @@ export interface Product {
   skuSpecifications: SkuSpecification[];
   sku: SingleSKU;
   titleTag: string;
+  deliveryPromisesBadges: BadgeItem[];
```

Comment:
 Shouldn't this be optional to avoid breaking [other projects](https://github.com/search?q=org%3Avtex-apps%20OR%20org%3Avtex%20%22%20from%20%27vtex.product-summary-context%27%22&type=code) that use Product type?

Reply:
Thanks, I'll change that!